features/admin.branding.v1/components/custom-text/custom-text-fields.tsx

```diff
@@ -155,6 +163,107 @@ const CustomTextFields: FunctionComponent<CustomTextFieldsProps> = (props: Custo
         );
     };
 
+    const handleMultiAttributeLink = (): void => {
+
+        const multiAttributeLoginConnectorHidden: boolean = serverConfigurationConfig.connectorsToHide
+            .includes(ServerConfigurationsConstants.MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID);
+
+        const alternativeLoginIdentifierHidden: boolean = serverConfigurationConfig.connectorsToHide
+            .includes(ServerConfigurationsConstants.ALTERNATIVE_LOGIN_IDENTIFIER);
+
+        if (!multiAttributeLoginConnectorHidden) {
+            history.push(
+                AppConstants.getPaths().get(
+                    "GOVERNANCE_CONNECTOR_EDIT"
+                ).replace(
+                    ":categoryId",
+                    ServerConfigurationsConstants.
+                        ACCOUNT_MANAGEMENT_CONNECTOR_CATEGORY_ID
+                ).replace(
+                    ":connectorId",
+                    ServerConfigurationsConstants.
+                        MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID
+                )
+            );
+        } else if (!alternativeLoginIdentifierHidden) {
+            history.push(AppConstants.getPaths().get("ALTERNATIVE_LOGIN_IDENTIFIER_EDIT"));
+        }
+    };
+
+    const getMultiAttributeLabel = (): string => {
+
+        const multiAttributeLoginConnectorHidden: boolean = serverConfigurationConfig.connectorsToHide
+            .includes(ServerConfigurationsConstants.MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID);
+
+        const alternativeLoginIdentifierHidden: boolean = serverConfigurationConfig.connectorsToHide
+            .includes(ServerConfigurationsConstants.ALTERNATIVE_LOGIN_IDENTIFIER);
+
+        if (!multiAttributeLoginConnectorHidden) {
+            return t("branding:connectors.multiAttributeLogin");
+        } else if (!alternativeLoginIdentifierHidden) {
+            return t("branding:connectors.alternativeLoginIdentifier");
+        }
+    };
+
+    const isMultiAttributeEnabled = (): boolean => {
+
+        return !serverConfigurationConfig.connectorsToHide
+            .includes(
+                ServerConfigurationsConstants.MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID
+            ) || !serverConfigurationConfig.connectorsToHide
+            .includes(
+                ServerConfigurationsConstants.ALTERNATIVE_LOGIN_IDENTIFIER
+            );
+    };
+
+    const getHelperText = (hintKey: string, warningKey: string, fieldName: string): ReactElement => {
+
+        return i18n.exists(hintKey) || i18n.exists(warningKey)
```

Comment:
better to return `null` first and avoid the turnary.

Reply:
Will address in a followup PR